cmd/kubeadm/app/phases/certs/certs.go

```diff
@@ -91,10 +241,6 @@ func NewServiceAccountSigningKey() (*rsa.PrivateKey, error) {
 }
 
 // NewFrontProxyCACertAndKey generate a self signed front proxy CA.
-// Front proxy CA and client certs are used to secure a front proxy authenticator which is used to assert identity
-// without the client cert.
-// This is a separte CA, so that front proxy identities cannot hit the API and normal client certs cannot be used
-// as front proxies.
 func NewFrontProxyCACertAndKey() (*x509.Certificate, *rsa.PrivateKey, error) {
```

Comment:
Nit: This func and `NewCACertAndKey` is identical; might be worth calling just `NewCACertAndKey` in the front-proxy code...